ngraph/frontend/tensorflow/src/op/bias_add.cpp

```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```

Comment:
this case is working  when bias input is static, it does not work for dynamic case. Need support for dynamic inputs